content/en/armory-enterprise/quick-spin-instructions/_index.md

```diff
@@ -39,8 +48,16 @@ The provided application deploys a basic NGINX instance and sets up a minimal de
 1. Click *Save Changes*.
 1. Select *Pipelines* for the left navigation menu and click *Start Manual Execution* for the `prepare-quick-spin-environment` pipeline.
```

Comment:
[Use bold for user interface elements](https://docs.armory.io/contribute/create/guides/docs-style-guide/#use-bold-for-user-interface-elements)